.env

```diff
@@ -1,6 +1,6 @@
-FAASM_VERSION=0.9.2
-FAASM_CLI_IMAGE=faasm/cli:0.9.2
-FAASM_WORKER_IMAGE=faasm/worker:0.9.2
+FAASM_VERSION=0.9.3
```

Comment:
I tag a new release to fix the errors in k8s files that the integration tests uncovered.